Gemfile

```
@@ -5,5 +5,5 @@ gemspec
 gem 'concurrent-ruby', '~> 1.0', require: 'concurrent'
 
 Dir[File.join(__dir__, 'bundler.d', '*.rb')].each do |bundle|
-  instance_eval(Bundler.read_file(bundle))
+  Bundler.eval_gemfile(bundle)
```

```
[!] There was an error parsing `Gemfile`: undefined method `eval_gemfile' for Bundler:Module. Bundler cannot continue.
```

You should be able to reproduce this locally with `bundle install`. This works for me
```suggestion
  eval_gemfile(bundle)
```

---

Yes, Thank you.